diff -ur c_icap-0.1.4.orig//c-icap.conf.in c_icap-0.1.4.mod//c-icap.conf.in
--- c_icap-0.1.4.orig//c-icap.conf.in	2010-05-29 17:36:42.000000000 +0200
+++ c_icap-0.1.4.mod//c-icap.conf.in	2011-03-21 10:41:03.879540951 +0100
@@ -9,8 +9,8 @@
 # Description:
 #	The file to store the pid of the main process of the c-icap server.
 # Default:
-#	PidFile /var/run/c-icap/c-icap.pid
-PidFile /var/run/c-icap/c-icap.pid
+#	PidFile /var/run/c-icap/c-icap.pid
+PidFile /var/run/c-icap/c-icap.pid
 
 # TAG: CommandsSocket
 # Format: CommandsSocket socket_file
@@ -119,6 +119,7 @@
 #	No value
 # Example:
 #	User wwwrun
+User	c-icap
 
 # TAG: Group
 # Format: Group groupname
@@ -128,7 +129,8 @@
 # Default:
 #	No value
 # Example:
-#	Group nogroup
+#	Group c-icap
+Group	c-icap
 
 # TAG: ServerAdmin
 # Format: ServerAdmin admin_mail
@@ -154,7 +156,7 @@
 #	dir is the location of temporary files.
 # Default:
 #	TmpDir /var/tmp
-TmpDir /var/tmp
+TmpDir /tmp
 
 # TAG: MaxMemObject
 # Format: MaxMemObject bytes
@@ -180,16 +182,16 @@
 # Description:
 #	The location of modules
 # Default:
-#	ModulesDir @prefix@/lib/c_icap
-ModulesDir @prefix@/lib/c_icap
+#	ModulesDir @libdir@/c_icap
+ModulesDir @libdir@/c_icap
 
 # TAG: ServicesDir
 # Format: ServicesDir dir
 # Description:
 #	The location of services
 # Default:
-#	ServicesDir @prefix@/lib/c_icap
-ServicesDir @prefix@/lib/c_icap
+#	ServicesDir @libdir@/c_icap
+ServicesDir @libdir@/c_icap
 
 # TAG: TemplateDir
 # Format: TemplateDir dir
@@ -221,8 +223,8 @@
 #	for more informations.
 #	It can be used more than once to use multiple magic files.
 # Default:
-#	LoadMagicFile @prefix@/etc/c-icap.magic
-LoadMagicFile @prefix@/etc/c-icap.magic
+#	LoadMagicFile /etc/c-icap/c-icap.magic
+LoadMagicFile /etc/c-icap/c-icap.magic
 
 # TAG: RemoteProxyUsers
 # Format: RemoteProxyUsers onoff
@@ -289,7 +291,7 @@
 # Default:
 #	No value
 # Example:
-#	basic_simple_db.UsersDB hash:/usr/local/c-icap/etc/c-icap-users.txt
+#	basic_simple_db.UsersDB hash:/etc/c-icap-users.txt
 
 # TAG: GroupSourceByGroup
 # Format: GroupSourceByGroup LookupTable
@@ -301,7 +303,7 @@
 # Default:
 #	No set
 # Example:
-#	GroupSourceByGroup hash:/usr/local/c-icap/etc/c-icap-groups.txt
+#	GroupSourceByGroup hash:/etc/c-icap-groups.txt
 
 # TAG: GroupSourceByUser
 # Format: GroupSourceByUser LookupTable
@@ -313,7 +315,7 @@
 # Default:
 #	No set
 # Example:
-#	GroupSourceByUser hash:/usr/local/c-icap/etc/c-icap-user-groups.txt
+#	GroupSourceByUser hash:/etc/c-icap-user-groups.txt
 
 # TAG: acl
 # Format: acl name type[{param}] value1 [value2] [...]
@@ -443,8 +445,8 @@
 #	store debugging information, errors and other
 #	information about the c-icap server.
 # Default:
-#	ServerLog @prefix@/var/log/server.log
-ServerLog @prefix@/var/log/server.log
+#	ServerLog /var/log/c-icap/server.log
+ServerLog /var/log/c-icap/server.log
 
 # TAG: AccessLog
 # Format: AccessLog LogFile [LogFormat] [[!]acl1] [[!]acl2] [...]
@@ -456,10 +458,10 @@
 #	This directive can be used more than once to specify more than
 #	one access log files
 # Default:
-#	AccessLog @prefix@/var/log/access.log
+#	AccessLog /var/log/c-icap/access.log
 # Example:
-#	AccessLog @prefix@/var/log/access.log MyFormat all
-AccessLog @prefix@/var/log/access.log
+#	AccessLog /var/log/c-icap/access.log MyFormat all
+AccessLog /var/log/c-icap/access.log
 
 # TAG: Logger
 # Format: Logger LoggerName
